webapi/src/main/scala/org/knora/webapi/routing/admin/UsersRouteADM.scala

```diff
@@ -49,10 +49,10 @@ final case class UsersRouteADM()(
       getUsersProjectMemberships() ~
       addUserToProjectMembership() ~
       removeUserFromProjectMembership() ~
-      getUsersProjectAdminMemberships() ~
+      getUsersProjectAdminMemberships ~
       addUserToProjectAdminMembership() ~
       removeUserFromProjectAdminMembership() ~
-      getUsersGroupMemberships() ~
+      getUsersGroupMemberships ~
```

Comment:
should probably be done everywhere or nowhere

Reply:
Maybe...I try to touch only what I need to refactor, to keep the history, as we spoke at some point. But I can improve it in the next PR if you like. I merged it already, because need the changes in current task, so to don't forget while I'm off...

Reply:
as you like - probably doesn't matter, just struck my eyes